source/installation.txt

```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
 
      - Description
 
-     - Reference
-
    * - Local
 
-     - Suitable for evaluation and proof of concept jobs.
+     - Suitable for evaluation and proof of concept jobs. 
 
-     - :ref:`Install on a Single Machine <single-machine-install>`
+       For more information, see :ref:`Install on a Single Machine 
+       <single-machine-install>`. 
 
    * - Unattended server
 
-     - Suitable for smaller production jobs.
-
-     - :ref:`Install on an Unattended Server <unattended-server>`
+     - Suitable for smaller production jobs. 
+       
+       For more information, see :ref:`Install on an Unattended Server 
+       <unattended-server>`. 
 
    * - Kafka
 
-     - Suitable for larger production jobs. Integrates with your own Kafka 
-       infrastructure to provide enhanced resiliency.
+     - The Kafka deployment method is recommended for long-running Change Data 
+       Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka 
+       is an open-source platform for distributed workloads that offers high 
+       availability and automatic recovery. This option is the most advanced, and is 
+       intended for users who require long-running or large production data syncs. 
+       
+       For more information, see :ref:`kafka-intro`.
+
+   * - Kafka with Confluent Cloud
+
+     - The Kafka with Confluent Cloud deployment method is an option for users who 
+       want the reliability of Kafka without having to manage their own cluster. 
+       Confluent is a fully-managed Kafka hosting provider run by the founders of 
+       Kafka. This deployment method uses APIs to add the newest Relational Migrator 
+       sink connectors to Confluent Cloud as managed connectors.
 
-     - The Kafka deployment model is not yet generally available, but you can 
-       reach out to your MongoDB account team to discuss private preview 
-       opportunities.
+       For more information, see :ref:`install-confluent-kafka`.
```

Comment:
[For reviewer]

This is causing a build error but will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.